### DISCUSSION TITLE
希望支持js和ts切换

### DISCUSSION THREAD

**Discussion by surge-squirrel** [element-plus/element-plus#13549] [Feature Request]
Feature type: **`TypeScript`**

## Details

- Enhanced component name: `All`

## Description

如题所示
很多组件库都支持这个功能
虽然vue3对ts有较好的支持，但是不代表普通开发者就会在卡法中采用ts
我很支持开源的库采用ts
谢谢

**Comment by kooriookami** [comment]
文档已更新
